moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -1238,7 +1402,24 @@ bool IKFastKinematicsPlugin::getPositionIK(const std::vector<geometry_msgs::Pose
   }
 
   KDL::Frame frame;
-  tf::poseMsgToKDL(ik_poses[0], frame);
+  if (transform_tip_ || transform_base_)
```

Comment:
this looks same as above - helper function?

Reply:
done